plugins/coverimage.koplugin/main.lua

```diff
@@ -472,13 +472,13 @@ function CoverImage:menuEntryCache()
                 text_func = function()
                     local number
                     if self.cover_image_cache_maxsize > 0 then
-                        number = self.cover_image_cache_maxsize
+                        number = self.cover_image_cache_maxsize .. "MiB"
```

Comment:
Perhaps you meant something like this? 
```suggestion
                        number = T(_("%1 MB"), self.cover_image_cache_maxsize)
```

NB Using MB (as in 1000 rather than 1024) because:
https://github.com/koreader/koreader/blob/86126d042f81e566006072ce310a5493cc403fae/frontend/util.lua#L939-L942

Reply:
Which really just means use the util func, btw. ;-)